docs/run_test/run_legacy.md

```diff
@@ -2,18 +2,26 @@
 
 [LISAv2](https://github.com/microsoft/lisa) brings incredible value to Linux
 quality on Microsoft virtualization platforms. To increase the value of LISA, we
-decide to continue invest and have more innovation on LISA. So, we started the
-next version.
+decide to continue to invest and have more innovation on LISA. So, we started
+the next version.
 
 During the transition time, we keep validating Linux kernels and distributions.
-We cannot and don't want to stop to wait the exciting current LISA. The two
+We cannot and don't want to stop to wait for the exciting current LISA. The two
 versions will be co-existing for a while. So, the current LISA version supports
 to invoke previous LISAv2.
 
 With this document, you will know how to run LISAv2 tests in the current LISA.
 The current LISA clones the LISAv2 repo, start LISAv2, and parse LISAv2 log to
 generate test results with new format.
 
+## :construction: WIP :construction:
+
+Attention: Currently we are migrating previous LISAv2 test cases to this LISA
+(v3) framework. Before we complete the test case migration, only T0 test cases
```

Comment:
v3 is used internally. We should avoid to mention it externally, or we can have an explanation about the "v3" in this document, and avoid to use it in other places.